src/protocols/rdp/channels/audio-input/audio-buffer.h

```diff
@@ -66,6 +66,29 @@ typedef struct guac_rdp_audio_format {
 
 } guac_rdp_audio_format;
 
+/**
+ * A structure of linked list to store the audio stream periodically
+ * sent by the client.
+ */
+typedef struct audio_stream_list audio_stream_list;
+struct audio_stream_list {
+    /**
+     * The pointer of audio stream
+     */
+    char* stream_data;
```

Comment:
Previously, this read:

```c
    /**
     * The data of audio buffer
     */
    char*  data;
```

From the previous review:

> The documentation of the structure and its members should clarify things at a high level. You say "the data of the audio buffer", but what _is_ that? What data?

I think the above still applies here. It is unclear what this structure member actually is. The documentation should clarify things at a high level.